client/components/event-create-page.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import CreateGameForm from './game-forms/CreateGameForm'
+class CreateGamePage extends Component {
+  onSubmit = newEvent => {
+    console.log('newEvent', newEvent)
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
Maybe wrap the form component in a "container"

Reply:
It depends whether this will be used as a sub-component or as root route.  Sub components will be inside a container already.  But yeah, if there is no parent implementing container we should probably give it className="container" here.